### DISCUSSION TITLE
Welcome to DevOps-Guide Discussions!

### DISCUSSION THREAD

**Discussion by Tikam02** [Tikam02/DevOps-Guide#139] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gregvgx** [comment]
Bonjour 
Comment vas-tu ?

**Comment by Gregvgx** [comment]
Je souhaiterais avoir des informations sur le développement d'un Dashboard avec la bibliothèque Dash et spyder.car quand je code sur spyder en voulant créer un Dashboard avec la bibliothèque dash ça ne fonctionne pas

**Comment by Njoxpy** [comment]
Hello devs

Does it require any technical background for diving into the world of DevOps?
